cockatrice/src/tappedout_interface.cpp

```diff
@@ -28,9 +28,17 @@ void TappedOutInterface::queryFinished(QNetworkReply *reply)
 
     if(reply->hasRawHeader("Location"))
```

Comment:
Can we qInfo log the response info - status code, location header, anything else relevant? This is useful information to capture into the user-visible debug log.